modules/javafx.controls/src/main/java/javafx/scene/control/skin/TreeTableRowSkin.java

```diff
@@ -97,10 +97,6 @@ public TreeTableRowSkin(TreeTableRow<T> control) {
 
         ListenerHelper lh = ListenerHelper.get(this);
 
-        lh.addChangeListener(control.indexProperty(), (ev) -> {
```

Comment:
Since https://github.com/openjdk/jfx/pull/1635 got merged, this is not needed as `TableRowSkinBase` does that already:
`registerChangeListener(control.indexProperty(), e -> requestCellUpdate());`. So this is basically a noop.